pages/common/fastmod.md

```diff
@@ -0,0 +1,21 @@
+# fastmod
+
+> A fast partial replacement for the codemod tool, replace and replace all in the whole codebase.
+> Regexes are match with the regex crate: <https://crates.io/crates/regex>.
+> More information: <https://github.com/facebookincubator/fastmod>.
+
+- Replace regex pattern:
+
+`fastmod {{regex_pattern}} {{replacement}} -d {{dir}} --iglob {{wildcard_patterns}}`
+
+- Replace fixed string as it is (not regex), .js or .json files only:
+
+`fastmod -F {{match_string}} {{replacement_string}} -e json,js`
+
+- Replace-all without confirmation, replace string as it is (not regex):
+
+`fastmod --accept-all -F {{match_string}} {{replacement_string}}`
+
+- Replace-all similar to the above, but print changed files as well:
```

Comment:
```suggestion
- Replace all like previous example, and print changed files:
```

Reply:
I want to add a multiline description for one of these examples to make it more clear, how to do that?

Reply:
Or say two sentences or one long rephrased sentence.

Reply:
> Replace regex pattern in current directory in all non-ignored files, you can ignore files with .ignore or .gitignore

Reply:
I just feel `and` is more fitting than `but` in this sentence. Separate this into 2 sentence is fine as well, like `Replace all similar to the above. And print changed files:`